core/templates/components/common-layout-directives/navigation-bars/side-navigation-bar.component.spec.ts

```diff
@@ -83,13 +109,92 @@ describe('Side Navigation Bar Component', () => {
     expect(componentInstance.currentUrl).toEqual(currentUrl);
   });
 
-  it('should toggle classroom submenu', () => {
-    componentInstance.classroomSubmenuIsShown = false;
-    componentInstance.toggleClassroomSubmenu();
-    expect(componentInstance.classroomSubmenuIsShown).toBeTrue();
+  it('should able to stop click propagation further', () => {
```

Comment:
The block does not have any expectations. Please validate if the click propagation was stopped

Reply:
done!